.github/workflows/nightly.yml

```
@@ -0,0 +1,412 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron:  '0 4 * * *'
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  DOCKER_BUILDKIT: 1
+  GOLANG_VERSION: 1.14
```

Optional:

Use same go version as in our Dockerfiles:
```suggestion
  GOLANG_VERSION: 1.15
```